test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -1889,10 +1819,7 @@ public void TensorFlowImageClassificationBadImages()
         public static string DownloadImageSet(string imagesDownloadFolder)
         {
             string fileName = "flower_photos_tiny_set_for_unit_tests.zip";
-            string url = $"https://mlnetfilestorage.file.core.windows.net/imagesets" +
-                $"/flower_images/flower_photos_tiny_set_for_unit_tests.zip?st=2019" +
-                $"-08-29T00%3A07%3A21Z&se=2030-08-30T00%3A07%3A00Z&sp=rl&sv=2018" +
-                $"-03-28&sr=f&sig=N8HbLziTcT61kstprNLmn%2BDC0JoMrNwo6yRWb3hLLag%3D";
+            string url = "https://tlcresources.blob.core.windows.net/datasets/flower_photos_tiny_set_for_unit_test.zip";
```

Comment:
What kind of images are not loading? AutoML should detect this issue and notify the user. Just swapping out the images means users will still run into this issue.

Reply:
The issue ended up not being the images, but the ordering. I have changed the dataset, so we use a short link now, but the images were not bad, just had some odd metadata, that I removed as I thought that may have contributed to the issue initially, however that ended up not being the issue.

---
In reply to: [347016790](https://github.com/dotnet/machinelearning/pull/4475#discussion_r347016790) [](ancestors = 347016790)

Reply:
Can we fix the loader such that a user doesn't run into the same issue?

Does the fix address the underlying issue? Or does fixing the dataset just avoid the issue for our unit test?

Reply:
Ahh... the order of the dataset rows differed between Windows/Unix, which lead to different dataset splits, which lead to differing metrics.

If we haven't, we may want to modify our example code where we load a folder of images.